packages/cashier/src/types/routes.types.ts

```diff
@@ -8,7 +8,7 @@ export type TRoute = {
     is_invisible?: boolean;
     path?: string;
     to?: string;
-    component?: typeof React.Component;
+    component?: React.FC<any>;
```

Comment:
Just curious why it's React.FC rather than React.Node?

Reply:
@shahzaib-deriv `React.ReactNode` is anything that React can render, It's not a component definition. The result of a component will be `React.ReactNode` or `JSX.Element`.